workers/unity/Packages/com.improbable.gdk.core/Logging/ForwardingDispatcher.cs

```diff
@@ -45,7 +45,22 @@ public void SetConnection(Connection connection)
 
         public void HandleLog(LogType type, LogEvent logEvent)
         {
-            Debug.unityLogger.Log(type, logEvent);
+            if (type == LogType.Exception)
+            {
+                // For exception types, Unity expects an exception object to be passed.
+                // Otherwise, it will not be displayed.
+                Exception exception = logEvent.Exception ?? new Exception(logEvent.ToString());
+
+                Debug.unityLogger.LogException(exception, logEvent.Context);
```

Comment:
This will cause a log to be sent twice to SpatialOS, once by this function and then also by LogCallback. Not sure how much of an issue this is though? My just mislead people.

Reply:
Double-sending is definitely something to avoid.

Reply:
Preventing double-sending with the newest commit